Gems/MotionMatching/Code/Source/TrajectoryQuery.cpp

```diff
@@ -21,39 +21,100 @@ namespace EMotionFX::MotionMatching
         return displacement;
     }
 
+    void TrajectoryQuery::PredictFutureTrajectory(const ActorInstance* actorInstance,
```

Comment:
nit: Might not be worth changing but if `actorInstance` should never be null (and null is not checked) consider making this a reference (same for `trajectoryFeature`)

Reply:
Changed